addon/styles/_frost-link.scss

```diff
@@ -121,6 +121,7 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
   &.tertiary {
     background-color: inherit;
     color: $frost-color-blue-1;
+    margin-bottom: -7px;
```

Comment:
I think that points to a need to solve this in a different way - maybe a missing baseline/vertical-align.  Either way, I don't think a fixed adjustment is going to work.